code/go/Makefile

```diff
@@ -19,6 +20,9 @@ check: lint check-license check-spec
 # which are not compliant with linter rules. The golint tool doesn't support ignore comments.
 lint:
 	@go list ./... | grep -v yamlschema | xargs -n 1 $(golint_cmd) -set_exit_status
+	@$(staticcheck_cmd) ./...
```

Comment:
Included in the same target `lint` following the example in `elastic-package` repository
https://github.com/elastic/elastic-package/blob/ddfa3a9027542a448e94d0e389563690e0c55c4b/Makefile#L33